samples/openthread/coap_client/README.rst

```diff
@@ -46,6 +49,14 @@ Button 2:
   Pressing results in sending a multicast LIGHT_ON or a multicast LIGHT_OFF message (alternatively) to the ``/light`` resource on server nodes.
   Sending this kind of message instead of a LIGHT_TOGGLE one allows to synchronize the state of the LEDs on several server nodes at once.
 
+Button 3:
+  Pressing results in switching power consumption modes: from SED to MED and from MED to SED.
+
+  .. note::
+     It is only active when the device is working as Minimal Thread Device (MTD) type.
+     It can be easly configurate with `overlay_sed.conf` applied for the build.
+     Defaultly OpenThread device is configured to run as Full Thread Device (FTD).
```

Comment:
I feel this information should be covered in the user guide and only linked here.
FTD default mode is mentioned there in Switching device type, but there is no mention of the `.conf` file.

Reply:
The user guide describes configuration options which are in fact the content of the `.conf` overlay file of this sample. The configuration overlay is just a convenience here.

Reply:
Then we should mention it in the user guide, and not mention it here. A link to user guide should be more than enough here.

Reply:
IMO: the overlay_sed.conf is example specific thing. I'd suggest adding general description how to pass multiple conf files to the application to ug_thread (if it is not mentioned in the general building documentation) and add info abut using "proj.conf" (default) to build FTD and "proj.conf overlay_sed.conf" in the building and running section. Sadly building app template is poor and does not support specifying configuration files so you'll need to add some nice text there keeping in mind that some people are using SES gui.

Reply:
ug_thread is not good place to inform about building details (passing many configuration files). It is already [here](http://developer.nordicsemi.com/nRF_Connect_SDK/doc/latest/zephyr/guides/kconfig/setting.html#initial-conf) In this sample README I would just mention that section.

Reply:
Sounds good!